### PR TITLE
Added ToolAdmin.restartPrimitiveTool to support immediate edit tools.

### PR DESCRIPTION
**Pull request by bbastings**

### CODE REVIEW

**Comment by bbastings** [comment]
@Josh-Schifter Your team will likely end up needing ToolAdmin.restartPrimitiveTool for immediate tools to create sheets, etc. Also tried to improve EditCommand documentation based on the meeting we had.

**Review by ben-polinsky — APPROVED** [review]

**Comment by Josh-Schifter** [comment]
Agreed on restartPrimitiveTool.  Doc improvements are appreciated.

**Review by Josh-Schifter — APPROVED** [review]

File: editor/backend/src/EditCommand.ts [hunk 1]
```diff
@@ -15,7 +15,21 @@ import { EditCommandIpc, EditorIpc, editorIpcStrings } from "@itwin/editor-commo
 export type EditCommandType = typeof EditCommand;
 
 /**
- * An EditCommand performs an editing action on the backend. EditCommands are usually paired with and driven by EditTools on the frontend.
+ * An EditCommand performs an editing action on the backend.
+ * Any writes to the iModel in an editing session should be done from an EditCommand:
+ * - All changes to an iModel are made within a transaction (Txn).
+ * - Calling SaveChanges ends the current Txn and starts a new one.
+ * - Using EditCommand ensures all of the changes in a Txn are from the same source, as only one EditCommand may be active at a time.
+ * - Because there is currently no way to enforce this, it is important that all applications follow this rule.
```
Comment (pmconne) [inline editor/backend/src/EditCommand.ts:23]: Is there an example EditCommand we can link to?

Comment (bbastings) [reply editor/backend/src/EditCommand.ts:23]: Only one, I could add "@see BasicManipulationCommand for an example" if you think that helps?

Comment (pmconne) [reply editor/backend/src/EditCommand.ts:23]: Sure.

Comment (bbastings) [reply editor/backend/src/EditCommand.ts:23]: done.

**Review by pmconne — APPROVED** [review]

**Pull request merged**